config.py

```diff
@@ -61,6 +61,10 @@ class Config:
     RAS_PARTY_SERVICE_PROTOCOL = os.getenv('RAS_PARTY_SERVICE_PROTOCOL', 'http')
     RAS_PARTY_SERVICE = f'{RAS_PARTY_SERVICE_PROTOCOL}://{RAS_PARTY_SERVICE_HOST}:{RAS_PARTY_SERVICE_PORT}/'
 
+    FRONTSTAGE_SERVICE_HOST = os.getenv('RAS_FRONTSTAGE_HOST', 'localhost')
+    FRONTSTAGE_SERVICE_PORT = os.getenv('RAS_FRONTSTAGE_PORT', 8082)
+    RAS_FRONTSTAGE_SERVICE = f'http://{FRONTSTAGE_SERVICE_HOST}:{FRONTSTAGE_SERVICE_PORT}/'
```

Comment:
Will need to split out the protocol (like the Party URL).  Localhost uses http, where preprod will hang unless you use https.

Reply:
Added protocol but also changed the env names so they match with the variable names 👍